typescript/sdk/src/deploy/core/HyperlaneCoreChecker.ts

```diff
@@ -171,12 +176,6 @@ export class HyperlaneCoreChecker extends HyperlaneAppChecker<
       contracts.interchainGasPaymaster.addresses,
       contracts.proxyAdmin.address,
     );
-    await this.checkProxiedContract(
-      chain,
-      'DefaultIsmInterchainGasPaymaster',
-      contracts.interchainGasPaymaster.addresses,
-      contracts.proxyAdmin.address,
-    );
```

Comment:
did we capture this case somewhere else?

Reply:
it's just not proxied